includes/results.inc

```diff
@@ -1021,7 +1021,11 @@ class IslandoraSolrFacets {
       $link['count'] = $count;
       $link['link_plus'] = '<a' . drupal_attributes($attributes['plus']['attr']) . '>+</a>';
       $link['link_minus'] = '<a' . drupal_attributes($attributes['minus']['attr']) . '>-</a>';
-      $buckets[] = $link;
+      $buckets[$bucket] = $link;
+    }
+
+    if (isset($this->settings['solr_field_settings']['sort_by']) && $this->settings['solr_field_settings']['sort_by'] == 'index') {
+      ksort($buckets);
```

Comment:
@adam-vessey I did that, but the problem was that you then still have to sort it this way as (for example) sorting by content model in solr you are sorting
```
islandora:compoundCModel
islandora:bookCModel
islandora:pageCModel
```
but then in Islandora you see the transformed text but still sorted based on the old text values.

I can go do what you are suggesting, but we still need to sort here to account for the later transformations.